src/applications/mhv-medical-records/containers/LandingPage.jsx

```diff
@@ -365,17 +378,38 @@ const LandingPage = () => {
                   data-testid="go-to-mhv-download-records"
                   className="vads-u-margin-bottom--2"
                 >
-                  <Link
-                    to="/download"
+                  <va-link-action
+                    href="/my-health/medical-records/download"
                     className="vads-c-action-link--blue"
                     data-testid="go-to-download-mr-reports"
-                    onClick={() => {
+                    text={MEDICAL_RECORDS_DOWNLOAD_LABEL}
+                    onClick={event => {
+                      event.preventDefault();
+                      history.push('/download');
```

Comment:
There is an inconsistency here. This and others above have a `history.push`, but on line 409 it does not. Why do we need a, `event.preventDefault()` and `history.push`? Find out why and see if the rest of the links can be made consistent.